backend/substrapp/tasks/tasks.py

```diff
@@ -797,9 +805,38 @@ def _do_task(client, subtuple_directory, tuple_type, subtuple, compute_plan_id,
     with open(path.join(pred_path, 'perf.json'), 'r') as perf_file:
         perf = json.load(perf_file)
     result['global_perf'] = perf['all']
+
+    tag = subtuple.get("tag")
+    if tag and TAG_VALUE_FOR_TRANSFERT_BUCKET in tag:
```

Comment:
as tag is a string, it should not be `TAG_VALUE_FOR_TRANSFERT_BUCKET == tag` or the `in` is done on purpose?

Reply:
Yes, it's on purpose. We want to match a potential sub-string here.

Reply:
👍